components/match2/wikis/callofduty/match_summary.lua

```diff
@@ -23,6 +23,11 @@ local NO_CHECK = '[[File:NoCheck.png|link=]]'
 
 local LINK_DATA = {
 	cdl = {icon = 'File:Call of Duty League Logo Small.png', text = 'Call of Duty League matchpage'},
+	breakingpoint = {
+		icon = 'File:Breaking Point GG icon lightmode.png',
```

Comment:
is there a higher res of it that could be provided on commons?

Reply:
are you talking about the one im adding or the COD League icon?

Reply:
`File:Breaking Point GG icon lightmode.png`

Reply:
As far as icons go? no. 
The best images i have are the full size image that these icons are based off of 
https://liquipedia.net/commons/index.php?search=Breaking+Point+GG&title=Special%3ASearch&profile=default&fulltext=1